Update README.md for blackwell and AutoModel (#12612)
NVIDIA/NeMo: README.md

<!-- markdownlint-disable -->
<details open>

>   <summary><b>Unveiling :hugs:Hugging Face models support via AutoModel</b></summary>

Reviewer: ```suggestion
  <summary><b>Pretrain and finetune :hugs:Hugging Face models via AutoModel</b></summary>
```

>       Nemo Framework's latest feature AutoModel enabled broad support for :hugs:Hugging Face model, with 25.02 focusing on <a href=https://huggingface.co/transformers/v3.5.1/model_doc/auto.html#automodelforcausallm>AutoModelForCausalLM<a> in the <a href=https://huggingface.co/models?pipeline_tag=text-generation&sort=trending>text generation category<a>. Future releases will enable broad support for more model families such as Vision Language Model.

Reviewer: ```suggestion
      Nemo Framework's latest feature AutoModel enables broad support for :hugs:Hugging Face models, with 25.02 focusing on <a href=https://huggingface.co/transformers/v3.5.1/model_doc/auto.html#automodelforcausallm>AutoModelForCausalLM<a> in the <a href=https://huggingface.co/models?pipeline_tag=text-generation&sort=trending>text generation category<a>. Future releases will enable support for more model families such as Vision Language Model.
```

</details>

<details open>
  <summary><b>Training on Blackwell using Nemo</b></summary>

>       Nemo Framework has added Blackwell support, with 25.02 focusing on functional parity for B200 and more optimizations in the upcoming releases.

Reviewer: ```suggestion
      NeMo Framework has added Blackwell support, with 25.02 focusing on functional parity for B200. More optimizations to come in the upcoming releases.
```
